site/content/docs/gsoc-2020-ideas.md

```diff
@@ -94,14 +112,31 @@ friendly front end could be created. This would allow a designer to select the
 signals they want to capture and the necessary logic analyser would be
 generated with all the required wiring to feed to signals into the analyser.
```

Comment:
You can add me as a mentor here, and you probably also want to add Greg.

Reply:
Done, thanks @luismarques .